adoc/ii-Strategy/SA.adoc

```diff
@@ -7,59 +7,45 @@ The strategy elements are typically used to model the strategic direction and ch
  
-Scalability & Flexibility:: With the cloud, you can scale up your system as and when required, add or remove resources when the business demands it and as mentioned above, "rightsizing" and instance optimization to provide an efficient solution.
-+    
-////
-With SUSE ready to run images available in the cloud providers marketplace it is easy to scale.
-////
+Scalability & Flexibility:: With the cloud and the ability to use pay-as-you-go subscriptions, businesses can scale up or scale down resources as needed.  It is much easier to "rightsize" and ensure an efficient environment that meets changing business demands.
 
-
-Maintenance:: With the public cloud, IT departments no longer have to worry about managing and maintaining the hardware and underlying infrastructure, the cloud service provider handles this for your organization and frees up company resources to focus on other business activities.
+Maintenance:: In the cloud, IT departments no longer have to worry about managing and maintaining the hardware and underlying infrastructure.  The cloud service provider handles this, freeing up company resources to focus on innovation and other business needs.
       
-Resiliency:: For businesses these days, uptime is of prime importance to ensure day-to-day business operations run smoothly. Moving to the cloud maximizes uptime and reduces downtime.
-The cloud improves disaster recovery and business continuity without the need to spend a huge amount of capital on robust disaster recovery tools.  Cloud providers offer a variety of services to help protect businesses from any security threat or outage.
-
-+
-SUSE's SAP HA Automation can add to these services to provide less downtime of your SAP application.
-    
-Remote access:: With the cloud, your employees can access data from anywhere, and at any time via the internet making business more flexible and increase the productivity of the employees.
-+    
-SUSE products natively provide many options for remote access and control.
-    
-So overall, public clouds offer significant benefits for all customers regardless of size. The use of public cloud resources can lower infrastructure costs and improve the scalability, agility, flexibility and availability of applications. 
+Resiliency:: Uptime is of prime importance to ensure day-to-day business operations run smoothly.  Moving to the cloud maximizes uptime and reduces downtime.
+The cloud improves disaster recovery and business continuity without the need to spend a huge amount of capital on robust disaster recovery tools.  Cloud providers offer a variety of services to help protect businesses from security threats and outages.  SUSE's SAP HA automation augments these services to further reduce downtime of SAP applications.
```

Comment:
similar here - we always have two layers : one is the cloud with the benefits, second how suse can make it better,etc. 
If you move it into one sentence its less highlighting...
But this is maybe only may view....